kraken/node_actions/aws_node_scenarios.py

```diff
@@ -90,6 +90,55 @@ def wait_until_terminated(self, instance_id, timeout=600):
             logging.error("Failed to get status waiting for %s to be terminated %s" % (instance_id, e))
             return False
 
+    # Creates a deny network acl and returns the id
+    def create_default_network_acl(self, vpc_id):
+        try:
+            logging.info("Trying to create a default deny network acl")
+            response = self.boto_client.create_network_acl(VpcId=vpc_id)
+            acl_id = response["NetworkAcl"]["NetworkAclId"]
+            logging.info("Created a network acl, id=%s" % acl_id)
+        except Exception as e:
+            logging.error("Failed to create the default network_acl: %s" % (e))
+            sys.exit(1)
+        return acl_id
+
+    # Replace network acl association
+    def replace_network_acl_association(self, association_id, acl_id):
+        try:
+            logging.info("Replacing the network acl associated with the subnet")
+            status = self.boto_client.replace_network_acl_association(AssociationId=association_id, NetworkAclId=acl_id)
+            logging.info(status)
+            new_association_id = status["NewAssociationId"]
+        except Exception as e:
+            logging.error("Failed to replace network acl association: %s" % (e))
+            sys.exit(1)
+        return new_association_id
+
+    # Describe network acl
+    def describe_network_acls(self, vpc_id, subnet_id):
+        try:
+            response = self.boto_client.describe_network_acls(
+                Filters=[
+                    {"Name": "vpc-id", "Values": [vpc_id]},
+                    {"Name": "association.subnet-id", "Values": [subnet_id]},
+                ]
+            )
+        except Exception as e:
+            logging.error("Failed to describe network acl: %s" % (e))
+        associations = response["NetworkAcls"][0]["Associations"]
```

Comment:
I had originally gotten the error message below. Might be nice to put this in a try catch as well and print off a message about making sure you have aws configured and set for the region of your vpc/subnet (which was my issue) 


```
  File "run_kraken.py", line 164, in main
    zone_outages.run(scenarios_list, config, wait_duration)
  File "/Users/prubenda/PycharmProjects/pplRepos/chaitanyaenr/kraken-1/kraken/zone_outage/actions.py", line 31, in run
    associations, original_acl_id = cloud_object.describe_network_acls(vpc_id, subnet_id)
  File "/Users/prubenda/PycharmProjects/pplRepos/chaitanyaenr/kraken-1/kraken/node_actions/aws_node_scenarios.py", line 128, in describe_network_acls
    associations = response["NetworkAcls"][0]["Associations"]

```

Reply:
Nice catch, included the message to make sure aws cli is configured right.